modules/test/playwright/tests/frontend-data-set-views-web/entries.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {loginTest} from '../../fixtures/loginTest';
+import {dataSetsPageTest} from './fixtures/dataSetsPageTest';
+
+export const test = mergeTests(dataSetsPageTest, loginTest);
+
+test('Assert table columns', async ({dataSetsPage, page}) => {
+	await dataSetsPage.goto();
+	await dataSetsPage.createSampleDataSetUI();
+
+	const tableColumns = await page
+		.locator('.dnd-thead > .dnd-tr')
+		.first()
+		.locator('.dnd-th')
+		.count();
+
+	await expect(tableColumns).toBe(6);
```

Comment:
@markocikos is this what you meant with checking the right amount of columns (rather than asserting the non-existence of the "Views" column)?

Imho it's more than fine as Kreso did. This table will receive changes soon so this is more than enough

Reply:
I had in mind a more thorough check of existing columns. In _addition_ to column count, we can check content of each expected header, something like six `getByText` checks. With those seven assertions, we can be sure that no other column is in the table.